pkg/hubble/parser/options/options_test.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 func TestRedact(t *testing.T) {
-	want := "level=info msg=\"configured Hubble with redact options\" options=\"&{CacheSize:3 HubbleRedactSettings:{Enabled:true RedactHTTPQuery:false RedactHTTPUserInfo:false RedactKafkaAPIKey:false RedactHttpHeaders:{Allow:map[] Deny:map[]}}}\"\n"
+	want := "level=info msg=\"configured Hubble with redact options\" options=\"&{CacheSize:3 HubbleRedactSettings:{Enabled:true RedactHTTPQuery:false RedactHTTPUserInfo:false RedactKafkaAPIKey:false RedactHttpHeaders:{Allow:map[] Deny:map[]}} EnableL3L4PolicyEnrichment:false}\"\n"
```

Comment:
I think we should change the redact log line to
```go
logger.Info("configured Hubble with redact options", logfields.Options, opt.HubbleRedactSettings)
```
That way the redact logging line only log redact related options. This will also require updating this line to
Something like
```suggestion
	want := "level=info msg=\"configured Hubble with redact options\" options=\"{Enabled:true RedactHTTPQuery:false RedactHTTPUserInfo:false RedactKafkaAPIKey:false RedactHttpHeaders:{Allow:map[] Deny:map[]}}\"\n"
```

Reply:
Makes sense. Updated!